### PR TITLE
[Enhancement] Support Slide Vertex TRT

### PR DESCRIPTION
**Pull request by grimoire**
```
DEPLOY_CFG_PATH="${MMDEPLOY_PATH}/configs/mmrotate/rotated-detection_tensorrt_dynamic-320x320-1024x1024.py"
MODEL_CFG_PATH="${MMROTATE_PATH}/configs/gliding_vertex/gliding_vertex_r50_fpn_1x_dota_le90.py"
CHECKPOINT_PATH="${TORCH_CHECKPOINT_PATH}/mmrotate/gliding_vertex_r50_fpn_1x_dota_le90-12e7423c.pth"
```

Add Sliding Vertex support. TensorRT only.

### CODE REVIEW

File: docs/en/03-benchmark/benchmark.md [hunk 1]
```diff
@@ -1641,6 +1641,18 @@ Users can directly test the performance through [how_to_evaluate_a_model.md](../
     <td align="center">-</td>
     <td align="center">-</td>
   </tr>
+  <tr>
```
Comment (RunningLeon) [inline docs/en/03-benchmark/benchmark.md:1644]: could add to regression test config for easy test: https://github.com/open-mmlab/mmdeploy/blob/master/tests/regression/mmrotate.yml

File: tests/test_codebase/test_mmrotate/test_mmrotate_models.py [hunk 2]
```diff
@@ -332,3 +332,222 @@ def test_get_bboxes_of_oriented_rpn_head(backend_type: Backend):
...
+    rewrite_inputs = {'x': x}
+    rewrite_outputs, is_backend_output = get_rewrite_outputs(
+        wrapped_model=wrapped_model,
+        model_inputs=rewrite_inputs,
+        deploy_cfg=deploy_cfg)
+    assert rewrite_outputs is not None
+
+
+@pytest.mark.parametrize('backend_type', [Backend.ONNXRUNTIME])
+def test_gv_ratio_roi_head__simple_test(backend_type: Backend):
```
Comment (RunningLeon) [inline tests/test_codebase/test_mmrotate/test_mmrotate_models.py:487]: failed with 
```

self = {'score_threshold': 0.05, 'iou_threshold': 0.1, 'pre_top_k': 2000, 'keep_top_k': 2000}, name = 'max_output_boxes_per_class'

    def __getattr__(self, name):
        try:
            value = super(ConfigDict, self).__getattr__(name)
        except KeyError:
            ex = AttributeError(f"'{self.__class__.__name__}' object has no "
                                f"attribute '{name}'")
        except Exception as e:
            ex = e
        else:
            return value
>       raise ex
E       AttributeError: 'ConfigDict' object has no attribute 'max_output_boxes_per_class'

../../../anaconda3/envs/pt1.8/lib/python3.7/site-packages/mmcv/utils/config.py:49: AttributeError
```

File: mmdeploy/codebase/mmrotate/models/dense_heads/rotated_rpn_head.py [hunk 3]
```diff
@@ -3,8 +3,9 @@
 
 from mmdeploy.codebase.mmdet import (get_post_processing_params,
                                      pad_with_value_if_necessary)
-from mmdeploy.codebase.mmrotate.core.post_processing import \
-    fake_multiclass_nms_rotated
+# from mmdeploy.codebase.mmrotate.core.post_processing import \
```
Comment (RunningLeon) [inline mmdeploy/codebase/mmrotate/models/dense_heads/rotated_rpn_head.py:6]: could we remove this line?

File: tests/regression/mmrotate.yml [hunk 4]
```diff
@@ -48,3 +48,21 @@ models:
...
+      - *pipeline_ort_detection_dynamic_fp32
+      - *pipeline_trt_detection_dynamic_fp32
+      - *pipeline_trt_detection_dynamic_fp16
+
+  - name: gliding_vertex
+    metafile: configs/gliding_vertex/metafile.yml
+    model_configs:
+      - configs/gliding_vertex/gliding_vertex_r50_fpn_1x_dota_le90.py
+    pipelines:
+      - *pipeline_ort_detection_dynamic_fp32
```
Comment (RunningLeon) [inline tests/regression/mmrotate.yml:66]: If `onnxruntime` is supported, we'd better update tables in `benchmark` and `suporrt_models` to `Y`

**Review by RunningLeon — APPROVED** [review]
LGTM

**Comment by RunningLeon** [comment]
@irexyc Hi, any comment on this PR?

**Review by irexyc — APPROVED** [review]

**Pull request merged**